kolibri/plugins/user_profile/assets/src/views/ChangeFacility/SelectFacility.vue

```diff
@@ -219,11 +220,13 @@
                 name: facility.name,
                 base_url: address.base_url,
                 address_id: address.id,
-                learner_can_sign_up: facility.learner_can_sign_up || true,
+                learner_can_sign_up: facility.learner_can_sign_up,
+                learner_can_login_with_no_password: facility.learner_can_login_with_no_password,
                 kolibri_version: address.kolibri_version,
               };
-              if (!this.availableFacilities.find(f => f.id === facility.id))
+              if (!this.availableFacilities.find(f => f.id === facility.id)) {
```

Comment:
Just added a little fix. I think that a true Python person wrote this `if` :))))